api/protocoordinatorservice/coordinator.proto

```diff
@@ -23,17 +23,27 @@ service Coordinator {
 }
 
 // A committer's representation of a block in the blockchain.
-message Block {
-    uint64 number = 1;                  // The block number.
-    repeated protoblocktx.Tx txs = 2;   // List of transactions within the block.
-    repeated uint32 txs_num = 3;        // Transaction number within the block generated by the orderer.
-    repeated TxStatusInfo rejected = 4; // Rejected transactions.
+message Batch {
+    repeated Tx txs = 1;                // List of transactions within the block.
+    repeated TxStatusInfo rejected = 2; // Rejected transactions.
+}
+
+// A committer's representation of a TX.
+message Tx {
+    TxId id = 1;
+    protoblocktx.Tx content = 2;
+}
+
+// The TX identifier with respect to the block generated by the orderer.
+message TxId {
```

Comment:
We can rename this TxID to `TxLocator`, or `TxRef`.

Reply:
`TxRef` sounds great.